### DISCUSSION TITLE
Is there a way to specify a default value for an import

### DISCUSSION THREAD

**Discussion by JAForbes** [open-policy-agent/community#389] [OPA and Rego] [answered]
E.g. something along the lines of:

`import input.currency = "USD"`

**Comment by anderseknert** [comment, marked as answer]
Not in the import statement itself. But you can assign it to a "normal" variable with a default value, which would do the same thing:

```rego
default currency := "USD"

currency := input.currency
```

**Reply by JAForbes** [reply]
Great thanks
